heathskin/game_state.py

```diff
@@ -106,10 +116,12 @@ def get_entity_by_name(self, ent_id, default=None):
         except ValueError:
             if ent_id == "GameEntity":
                 result_id = "1"
-            elif ent_id == self.friendly_user_name:
-                result_id = "2"
+            elif hasattr(self,'player1') and  ent_id == self.player1:
+                result_id = "2" if self._is_player_first() else "3"
+            elif hasattr(self, 'player2') and ent_id == self.player2:
+                result_id = "3" if self._is_player_first() else "2"
             else:
-                result_id = "3"
+                print 'failed to get entity by name : %s' % ent_id
```

Comment:
So I am seeing plenty of lines like this 
<code>
TAG_CHANGE Entity=[name=Gul'dan id=36 zone=PLAY zonePos=0 cardId=HERO_07 player=2] tag=NUM_TURNS_IN_PLAY value=1
</code>
which would his this exception case.. is this entity really entity 36?  does this line need further parsing?